site/content/agent-app-widgets/introduction.md

```diff
@@ -4,13 +4,15 @@ weight: 10
 
 # Introduction
 
-Agent App Widgets are web applications loaded inside the LiveChat Agent App. All agents can interact with the widget during chats with customers. The widget itself is displayed in the Agent's App sidebar:
+There are several ways to extend the integration interface of the **LiveChat Agent App**. The most common one makes use of widgets. **Agent App Widgets** are web applications loaded inside the LiveChat Agent App. 
+
+Such a solution allows for interactions between widgets and agents (people chatting with customers). For example, agents could benefit from analytics, collected and displayed by a widget.
 
 <img src="../assets/images/agent-app-widget-sample.png" width="600" style="margin-top: 20px;max-width: 100%;"/>
 
-## Overview
+Typically, widgets are displayed in the right sidebar, as a part of the **Details** section. This is not the only possible placement for widgets, though. Other placement options include **Settings** and [Fullscreen Apps](#sample-apps).
```

Comment:
`and [Fullscreen Apps](#sample-apps)` - should it point to `#sample apps`?

Reply:
done